src-electron/generator/helper-zcl.js

```diff
@@ -2614,8 +2466,6 @@ exports.if_manufacturing_specific_cluster = dep(
   if_manufacturing_specific_cluster,
   { to: 'if_mfg_specific_cluster' }
 )
-exports.zcl_command_argument_type_to_cli_data_type =
-  zcl_command_argument_type_to_cli_data_type
```

Comment:
do we need to maintain backward compatibility for the zcl_command_argument_type_to_cli_data_type api under helper-zcl.js?

Reply:
This helper is still present and generates the correct content in helper-zigbee-zcl.js